src/wmnds/assets/sass/_vars.scss

```diff
@@ -4,6 +4,7 @@
 $palettes: (
   primary: #3c1053,
   secondary: #9d5baf,
+  primarydark: #1e082a,
```

Comment:
Can we get this shade with the darker instead? Using it like that:  background-color: get-color(primary, 20);
Ask @jkmnch if this is shade or a new color and maybe he can give you the percentage that you should be using.

Reply:
The sign-in banner colour is Primary Purple but 50% darker. Since it's darker does it need to have its own hex code, or can it still be made darker using the code catia has suggested